appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-document/controller/date-submitted.js

```diff
@@ -0,0 +1,12 @@
+import { postDateSubmittedFactory, renderDateSubmittedFactory } from '../../common/index.js';
+
+export const get = renderDateSubmittedFactory({
+	getBackLinkUrl: (appealDetails, comment) =>
+		`/appeals-service/appeal-details/${appealDetails.appealId}/interested-party-comments/${comment.id}/add-document/redaction-status`
+});
+
+export const post = postDateSubmittedFactory({
```

Comment:
In keeping with the discussion today in the dev channel re. feature filenames including the feature name as a prefix, I think for consistency, these should also be fully-qualified names (eg. `postDateSubmitted`) like those in other feature code